eng/pipelines/templates/steps/net-release-to-any.yml

```diff
@@ -0,0 +1,140 @@
+parameters:
+  DownloadArtifacts: false
+  CreateGitTagAndRelease: false
+  CopyFromPartnerDrops: false
+  SignPackages: false
+  PublishSymbols: false
+  PublishToDevOpsFeed: false
+  PublishArtifact: false
+  PublishToNuget: false
+  PublishToBlobFeed: false
+  CopySignedFilesToBlob: false
+  LinkToSignedPackages: false
+  BuildToolsRepoPath: ''
+  ArtifactsPath: ''
+  DevOpsFeedId: ''
+  BlobFeedUrl: ''
+  PartnerDropsBlobPath: ''
+  PartnerDropsStorageName: ''
+  PartnerDropsContainerName: ''
+  PartnerDropsBlobPrefix: ''
+  PartnerDropsSubscription: ''
+  SignedPackagesLink: ''
+  NugetVersion: ''
+  CheckoutDirectory: $(Build.SourcesDirectory)
+
+steps:
+  - ${{ if ne(parameters.BuildToolsRepoPath, '') }}:
+    - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
+      parameters:
+        BuildToolsRepoPath: ${{ parameters.BuildToolsRepoPath }}
+
+  - ${{ if eq(parameters.DownloadArtifacts, 'true') }}:
+    - download: none # Disable the automatic downloading so that we can download ourselves to the location we want.
+    - task: DownloadPipelineArtifact@2
+      displayName: Download Packages
+      inputs:
+        artifact: packages
+        path: ${{ parameters.ArtifactsPath }}
+      condition: and(succeeded(), ne(variables['SkipDownloadArtifacts'], 'true'))
+
+  - ${{ if eq(parameters.CopyFromPartnerDrops, 'true') }}:
+    - task: PowerShell@2
+      displayName: 'Copy from AzureSdkPartnerDrops'
+      inputs:
+        targetType: filePath
+        filePath: '${{ parameters.BuildToolsRepoPath }}/scripts/copy-from-azuresdkpartnerdrops.ps1'
+        arguments: >
+          -downloadDir ${{ parameters.ArtifactsPath }}
+          -blobPath ${{ parameters.PartnerDropsBlobPath }}
+          -blobKey $(azuresdkpartnerdrops-access-key)
+      condition: and(succeeded(), ne(variables['SkipCopyFromPartnerDrops'], 'true'))
+
+  - ${{ if eq(parameters.CreateGitTagAndRelease, 'true') }}:
+    - task: PowerShell@2
+      displayName: 'Verify Package Tags and Create Git Releases'
+      inputs:
+        pwsh: true
+        targetType: filePath
+        filePath: '${{ parameters.CheckoutDirectory }}/eng/common/scripts/create-tags-and-git-release.ps1'
+        arguments: >
+          -artifactLocation ${{ parameters.ArtifactsPath }}
+          -workingDirectory $(System.DefaultWorkingDirectory)
+          -packageRepository Nuget
+          -releaseSha $(Build.SourceVersion)
+          -repoId $(Build.Repository.Id)
+      env:
+        GH_TOKEN: $(azuresdk-github-pat)
+      condition: and(succeeded(), ne(variables['SkipCreateGitTagAndRelease'], 'true'))
+
+  - ${{ if eq(parameters.SignPackages, 'true') }}:
+    - template: pipelines/steps/net-signing.yml@azure-sdk-build-tools
+      parameters:
+        PackagesPath: ${{ parameters.ArtifactsPath }}
+        BuildToolsPath: ${{ parameters.BuildToolsRepoPath }}
+
+  - ${{ if eq(parameters.PublishSymbols, 'true') }}:
```

Comment:
What is the goal by moving these templates into this repo? For things we are we are depending on things in the azure-sdk-build-tools repo already we should just keep those things over there.

Reply:
Well, I felt since this is specific to dotnet and I plan to move accomplish all the other UI net - release pipelines (Management, MultiAPI, fluent-SDK, and PartnerDrops) through this template it is best to just put it in the dotnet repo.

Reply:
Lets chat more about your plans tomorrow in our 1:1